### PR TITLE
Fiberzap: Adapter for fiber/log

### PR DESCRIPTION
**Pull request by Skyenought**
This update is to adapt the fiberlog package

## Example

```go
package main

import (
	"context"

	"github.com/gofiber/contrib/fiberzap"
	"github.com/gofiber/fiber/v2"
	"github.com/gofiber/fiber/v2/log"
)

func main() {
	app := fiber.New()
	log.SetLogger(fiberzap.NewLogger(fiberzap.LoggerConfig{
		ExtraKeys: []string{"request_id"},
	}))
	app.Use(func(c *fiber.Ctx) error {
		ctx := context.WithValue(c.UserContext(), "request_id", "123")
		c.SetUserContext(ctx)
		return c.Next()
	})
	app.Get("/", func(c *fiber.Ctx) error {
		log.WithContext(c.UserContext()).Info("Hello, World!")
		log.Infow("Hello, World!", "test", "custom")
		log.Infof("Hello, %s", "World!")
		return c.SendString("Hello, World!")
	})
	log.Fatal(app.Listen(":3000"))
}

```
result:
```shell
{"level":"info","ts":1688009051.6242049,"msg":"Hello, World!","request_id":"123"}
{"level":"info","ts":1688009051.6242645,"msg":"Hello, World!","test":"custom"}
{"level":"info","ts":1688009051.6242726,"msg":"Hello, World!"}
```

### CODE REVIEW

**Review by efectn — APPROVED** [review]

File: fiberzap/go.mod [hunk 1]
```diff
@@ -3,21 +3,24 @@ module github.com/gofiber/contrib/fiberzap
 go 1.18
 
 require (
-	github.com/gofiber/fiber/v2 v2.47.0
+	github.com/gofiber/fiber/v2 v2.47.1-0.20230626061657-fefc5338347d
```
Comment (ReneWerner87) [inline fiberzap/go.mod:6]: lets wait for the next release

Comment (Skyenought) [reply fiberzap/go.mod:6]: ok

**Comment by ReneWerner87** [comment]
- [ ] everything looks good, we are just waiting for the next fiber release

**Comment by gaby** [comment]
@Skyenought The version of Fiber is out with support for Logger interface. You can rebase/update the PR now 😁

**Comment by ReneWerner87** [comment]
either you change the go.mod yourself or wait for dependabot and update with the master afterwards

File: fiberzap/go.mod (hunk 1 above)
Comment (ReneWerner87) [reply fiberzap/go.mod:6]: new version is v2.48.0
https://github.com/gofiber/fiber/releases/tag/v2.48.0

**Comment by gaby** [comment]
@Skyenought Small conflict with main

**Review by gaby — APPROVED** [review]

**Pull request merged**